.travis/setup_binfmt_misc.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash -ex
+
+echo ':arm:M::\x7fELF\x01\x01\x01\x00\x00\x00\x00\x00\x00\x00\x00\x00\x02\x00\x28\x00:\xff\xff\xff\xff\xff\xff\xff\x00\xff\xff\xff\xff\xff\xff\xff\xff\xfe\xff\xff\xff:/usr/local/bin/qemu-arm-static:' > /proc/sys/fs/binfmt_misc/register
```

Comment:
Also, that means I have to re-evaluate drone.io as I didn't realized binfmt_misc wasn't needed at all anymore.